docs/crr-to-public-cloud.md

```diff
@@ -0,0 +1,123 @@
+# CRR from CloudServer to public cloud - Workflow
+
+## Installation
+
+An endpoint ('awsbackend', for example) will be defined in the [location
+configuration](https://github.com/scality/S3/blob/master/locationConfig.json) of
+[CloudServer](https://github.com/scality/S3). Each location constraint in the
+configuration file has properties needed for performing operations for the
+location. For example, if replicating to AWS S3, the properties will include the
+bucket name, region, and credentials for AWS S3. This enables the multiple
+backend feature in CloudServer that Backbeat uses for CRR to a public cloud.
+
+Replication endpoints are defined in the config for both Backbeat and
+CloudServer in Federation's [group_vars/all](https://github.com/scality/
```

Comment:
Federation is EE only... do we have something to point open source users to? if ot's the paragraph below, can we add templates, like for Federation?